src/packages/cli/src/CLI.ts

```diff
@@ -135,9 +150,14 @@ export class CLI implements Command {
 
       Generate artifacts (e.g. Prisma Client)
       ${chalk.dim('$')} prisma generate
+
+      Browse your data
+      ${chalk.dim('$')} prisma studio
+
+      Push the Prisma schema state to the database
+      ${chalk.dim('$')} prisma db push --preview
```

Comment:
Same as https://github.com/prisma/prisma/pull/4027/files#r511902014

Reply:
As discussed, product decided to not hide commands in the Preview state